modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/imageio/netcdf/utilities/NetCDFUtilities.java

```diff
@@ -453,7 +452,7 @@ public static int getDimensionLength(Variable var, final int dimensionIndex) {
      */
     public static int getRawDataType(final VariableIF variable) {
         VariableDS ds = (VariableDS) variable;
-        final DataType type = ds.getOriginalDataType();
+        final DataType type = ds.getDataType();
```

Comment:
It doesn't now, but it probably should. If you're not enhancing the variables then you probably want the original data type. I'll change this to check for the ScaleMissing system property.